src/com/epam/homework/maps/studentcounter/Main.java

```diff
@@ -0,0 +1,25 @@
+package com.epam.homework.maps.studentcounter;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    public static void main(String[] args) {
+        List<Student> list = initStudentList();
+        Counter counter = new Counter();
+        System.out.println(counter.count(list));
+        System.out.println(counter.countStudentsInFaculties(list));
```

Comment:
In case there is no student in a Faculty, will this print the number of students in that Faculty?

Reply:
No, method only prints counts for those faculties that have students, it receives students as input  and calculates counts based on their faculties. Should I add method that counts for all faculties?

Reply:
Just change current implementation to print 0-s in such cases

Reply:
Changed this method, now it includes faculties with 0 count of students